<html xmlns:py="http://genshi.edgewall.org/" xmlns:xi="http://www.w3.org/2001/XInclude" py:strip="">

<py:def function="announcement_id">
PMASA-2011-20
</py:def>

<py:def function="announcement_date">
2011-12-21
</py:def>

<py:def function="announcement_summary">
XSS in export.
</py:def>

<py:def function="announcement_description">
Using crafted url parameters, it was possible to produce XSS on the export panels in the server, database and table sections.
</py:def>

<py:def function="announcement_mitigation">
These attacks are unlikely to succeed on a victim. Moreover, all these attacks require that the user be already logged in and that a valid token be part of the request.
</py:def>

<py:def function="announcement_severity">
We consider these vulnerabilities to be non critical.
</py:def>

<py:def function="announcement_affected">
Versions 3.4.x are affected.
</py:def>

<py:def function="announcement_solution">
Upgrade to phpMyAdmin 3.4.9 or newer or apply the related patches listed below.
</py:def>

<py:def function="announcement_references">
Thanks to <a href="https://twitter.com/totally_unknown">Nils Juenemann</a> for reporting a vulnerable url parameter.
</py:def>

<py:def function="announcement_cve">CVE-2011-4780</py:def>

<py:def function="announcement_cwe">661 79</py:def>

<py:def function="announcement_commits">
bd3735ba584e7a49aee78813845245354b061f61
</py:def>

<xi:include href="_page.tpl" />
</html>


